ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-base.md

```diff
@@ -88,16 +88,18 @@ interface Management1
 
 | BGP AS | Router ID |
 | ------ | --------- |
-| 65101|  - |
+| 65101|  192.168.255.3 |
```

Comment:
You fixed this extra space in the last commit. But might have forgot to update the artifacts.

Reply:
Tony, this is not coming from my code. Its from here:

```
| BGP AS | Router ID |
| ------ | --------- |
| {{ router_bgp.as }}|  {{ router_bgp.router_id | arista.avd.default('-') }} |
{%     if router_bgp.bgp_cluster_id is arista.avd.defined %}
```

Reply:
Got you. Thank you! My bad I was confused, how did the CI test pass.